src/Explore.tsx

```diff
@@ -109,7 +109,7 @@ export default class Explore extends React.Component<{}, ExploreState> {
           isSelected={isSelected}
           handleChanges={handleChanges}
         />
-        <h3 className="SRC-boldText">
+        <h3 id="exploreCount" className="SRC-boldText">
```

Comment:
no way for multiple Explore components to be on the same page?

Reply:
correct! just one will show